build/docs.js

```diff
@@ -6,8 +6,12 @@ const { glob } = require("glob");
 const fm = require("front-matter");
 const mk = require("marked");
 
-async function buildDocs() {
-  glob(path.resolve(__dirname, "../src/docs/**/*.md"), async (err, files) => {
+async function mdToJson(mdDir, jsonFile){
+  if(!mdDir.endsWith("/")){
```

Comment:
Instead of this you can just use nodes `path.join`, it will add all of the intervening `/` in the output string